test/tests/val-2a/val-2a_TMAP4.i

```diff
@@ -0,0 +1,81 @@
+nx_scale = 5
+
+[Mesh]
+  [cartesian_mesh_TMAP4]
+    type = CartesianMeshGenerator
+    dim = 1
+    #     num
+    dx = '${fparse 5 * ${units 4e-9 m -> mum}}  ${units 1e-8 m -> mum}  ${units 1e-7 m -> mum}
+          ${units 1e-6 m -> mum}                ${units 1e-5 m -> mum}  ${fparse 10 * ${units 4.88e-5 m -> mum}}'
+    ix = '${fparse 5 * ${nx_scale}}             ${nx_scale}             ${nx_scale}
+          ${nx_scale}                           ${nx_scale}             ${fparse 10 * ${nx_scale}}'
+  []
+[]
```

Comment:
I see that you are trying to replicate the mesh from TMAP4. Why not just create a better mesh with TMAP8's capabilities? We don't need to exactly replicate TMAP4, we want to use our more elaborate capabilities to do even better whenever possible.